part4/performance-tuning.md

```diff
@@ -28,6 +28,12 @@ If you have generated the tables automatically using your JPA implementation (e.
 -   Sagas
 
     Table 'AssociationValueEntry', columns `associationKey` and `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaType` and `sagaId`,
```

Comment:
I agree on your first statement.

I would say that the index on the value would be of more benefit due to high cardinality.